Microsoft.DotNet.Interactive.PowerShell/Commands/TracePipelineObjectCommand.cs

```diff
@@ -20,7 +20,7 @@ public sealed class TracePipelineObjectCommand : PSCmdlet
         /// <summary>
         /// The object from pipeline.
         /// </summary>
-        [Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true)]
+        [Parameter(Position = 0, ValueFromPipeline = true)]
```

Comment:
What's wrong with having the `Mandatory` here?

Reply:
If I have a cell that contains just `$thisVarNotDefined` an error is written out saying `trace-pipelineobject 's InputObject parameter is mandatory and can't handle null`

Reply:
I see. Then we should add `[AllowNull]` instead of removing `Mandatory = true`.

Reply:
added!